carla/visualisation/output.py

```diff
@@ -17,5 +19,5 @@ def cprint(color, text, **kwargs):
         "c": "36",
         "w": "37",
     }
-    print("\x1b[%s%sm%s\x1b[0m" % (pre_code, code[color], text), **kwargs)
+    log.info("\x1b[%s%sm%s\x1b[0m" % (pre_code, code[color], text), **kwargs)
```

Comment:
Does that work with the logging aswell? Also: Where do we use cprint again?

Reply:
Worked in my tests 😄 

This `cprint` method is originated from the CLUE legacy 😅 and only used there. We decided to keep it at the moment and make it accessible for everyone. But I'd be also be happy to remove this.

Reply:
Since we have logging now, would be happy to replace the cprint annoyance and replace completly with logs